Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:latest
+
```

Comment:
use the alpine based openjdk images please we don't need this large of an image

Reply:
`openjdk:8-jdk-alpine`